setup-scripts/install.sh

```diff
@@ -81,15 +121,106 @@ msg "Installing node.js@8.11" $WHITE
 nvm install 8.11 --latest-npm
 
 if [ "$(command -v docker)" == "" ]; then
```

Comment:
@bradleystachurski for each section of the install script, we have a comment that is like `# Ensure nvm is installed` or `#install nodejs`, we should make sure we have comments for each of the docker sections.

Example: `# Checking docker installation versions` etc..